README.md

```diff
@@ -20,24 +20,24 @@
 <hr/>
 
 
-## Auto-DL is just DL but without code.
+## Auto-DL is just DL but without code.🤩
 
 Building deep learning models may take a lot of time. But not with AutoDL.  
-Automate your **meta-work** so that you can focus on building your models without the hassle of writing code.
+✨Automate your **meta-work** so that you can focus on building your models without the hassle of writing code.✨
```

Comment:
Please remove the emojis from this line
```suggestion
Automate your **meta-work** so that you can focus on building your models without the hassle of writing code.
```

Reply:
> Please remove the emojis from this line

Done sir